sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ldevice/LDeviceAdapter.java

```diff
@@ -514,4 +516,15 @@ public List<ExtRefInfo.ExtRefBayReference> getExtRefBayReferenceForActifLDEPF(fi
         return extRefBayReferenceList;
     }
 
+    private TFCDA toFCDA(TFCDAFilter tfcdaFilter) {
```

Comment:
Utility function for creating SCL related objects can take place in SclConstructorHelper